dtls_debug.c

```diff
@@ -127,31 +138,43 @@ dsrv_print_addr(const session_t *addr, char *buf, size_t len) {
       return 0;
 
     *p++ = '[';
+    --len;
 
     addrptr = &addr->addr.sin6.sin6_addr;
     port = ntohs(addr->addr.sin6.sin6_port);
 
     break;
   default:
-    memcpy(buf, "(unknown address type)", min(22, len));
-    return min(22, len);
+    // include terminating \000
```

Comment:
Please use the `/* ... */` comment syntax.

Reply:
Yes, I will do. (Sorry for that, I work on too much stuff in parallel with all different rules. Please keep on requesting to comply to your source format rules.)

Reply:
Done